modules/InboundEmail/InboundEmail.php

```diff
@@ -5121,38 +5121,64 @@ public function returnImportedEmail($msgNo, $uid, $forDisplay = false, $clean_em
 
     /**
      * Used to view non imported emails
-     * @param $msgNo - imap mesgno
-     * @param $uid - imap uid
-     * @return bool|Email - false on error | a non imported email
+     * @param string $msgNo - imap mesgno
+     * @param string $uid - imap uid
+     * @return Email|boolean - false on error | a non imported email
+     * @throws Exception
      */
     public function returnNonImportedEmail($msgNo, $uid)
     {
-        global $timedate;
-        global $app_strings;
-        global $app_list_strings;
-        global $sugar_config;
-        global $current_user;
+        global $timedate, $current_user;
 
         if (empty($header)) {
             $email = new Email();
 
 
             $this->connectMailserver();
+
+
             $header = imap_fetch_overview($this->conn, $uid, FT_UID);
             $fullHeader = imap_fetchheader($this->conn, $uid, FT_UID);
             $headerByMsgNo = imap_headerinfo($this->conn, $msgNo);
             $structure = imap_fetchstructure($this->conn, $uid, FT_UID); // map of email
             $email->name = $this->handleMimeHeaderDecode($header[0]->subject);
             $email->type = 'inbound';
-            if (!empty($unixHeaderDate)) {
-                $email->date_sent = $timedate->asUser($unixHeaderDate);
-                list($email->date_start, $email->time_start) = $timedate->split_date_time($email->date_sent);
 
-            } else {
-                $email->date_start = $email->time_start = $email->date_sent = "";
-            }
 
+            if(empty($email->date_entered)) {
+
+                // find if string has (UTC) in timezone
+                $pattern = '/([\w-+\/]+)/i';
```

Comment:
use preg escape char before brackets `/\([\w-+\/]+\)/i` if you want to find it in the pattern.